src/constants/notification-content.js

```diff
@@ -0,0 +1,167 @@
+export default function getNotificationContent({ type, payload }) {
+  let fragments;
+  switch (type) {
+    case "purchase_status_changed": {
```

Comment:
`case`s do not need curly braces. It is only affecting readability since objects are used a lot inside them

Reply:
Yeah, they don't, but I find it easier to read. This way the blocks are clearly defined